src/localization/dispensingStrings.json

```
@@ -74,10 +76,15 @@
     "patients": "Patients",
     "payment_amount": "Montant du paiement",
     "payment": "Paiement",
+    "payments": "Paiments",
+    "receipts": "Reçus",
     "prescriber": "Prescripteur",
     "prescribers": "Prescripteurs",
     "select_a_policy": "Sélectionnez une police d'assurance",
     "select_items": "Sélectionnez des articles",
+    "select_a_payment_type": "sélectionnez un type de paiement",
```

```suggestion
    "select_a_payment_type": "Sélectionnez un type de paiement",
```

and UPPERCASE first letter for the next two too?

---

6f89e6d